lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -531,6 +538,11 @@ public static function invalidDiscriminatorColumnType($className, $type)
         return new self("Discriminator column type on entity class '$className' is not allowed to be '$type'. 'string' or 'integer' type variables are suggested!");
     }
 
+    public static function invalidDiscriminatorValue($className)
+    {
+        return new self("Discriminator value on entity class '$className' should be a valud 'string'");
```

Comment:
There is a typo in `valud`. 😛

Reply:
Don't think this way. We're mainly talking about potentially including this in 3.0